### PR TITLE
fix ContextMenu FlyOut position on initial render

### LINKED ISSUE #296: ContextMenu Flyout Incorrectly positioned on initial render

### PR DESCRIPTION
**Pull request by sodiumjoe**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] ~~Two~~ One core team engineer approval~~s~~

fixes #296

### CODE REVIEW

File: src/components/ContextMenu/ContextMenu.less [hunk 1]
```diff
@@ -4,4 +4,12 @@
...
 .lucid-ContextMenu-FlyOut {
 	z-index: 300;
+	position: absolute;
+	// On first render, we don't have a an element to calculate measurements
+	// against so we just hide it and position at origin with max-height so it
+	// doesn't affect layout. We override all of these after initial render
+	opacity: 0;
+	top: 0;
+	left: 0;
+	max-height: 100vh;
```
Comment (jondlm) [inline src/components/ContextMenu/ContextMenu.less]: I'm curious why display none didn't achieve this same result?

File: src/components/ContextMenu/ContextMenu.jsx [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { createClass, findTypes } from '../../util/component-types';
 import { getAbsoluteBoundingClientRect } from '../../util/dom-helpers';
 import { lucidClassNames } from '../../util/style-helpers';
 
+var count = 0;
```
Comment (jondlm) [inline src/components/ContextMenu/ContextMenu.jsx]: Extraneous?

Comment (sodiumjoe) [inline src/components/ContextMenu/ContextMenu.jsx]: whoops, yeah, that was for testing something

File: src/components/ContextMenu/ContextMenu.less (hunk 1 above)
Comment (sodiumjoe) [inline src/components/ContextMenu/ContextMenu.less]: @alisonailea suggested this for perf reasons, I think

Comment (sodiumjoe) [inline src/components/ContextMenu/ContextMenu.less]: i think `display: none` also doesn't render the element properly to make calculations from

**Comment by chrisdopuch** [comment]
LGTM, thanks @jmoon :)

**Pull request merged**